docs/notebooks/introduction.rst

```diff
@@ -184,4 +189,41 @@ You can download a specific Notebook checkpoint from Neptune to Jupyter.
   - Checkpoint
 
 
-3. Click **Download**.
\ No newline at end of file
+3. Click **Download**.
+
+Using the Neptune CLI
```

Comment:
Two general points:

1. The CLI assumes that `NEPTUNE_API_TOKEN` environment variable is available. This is very important, there's no way around it. It also respects `NEPTUNE_PROJECT` - if you have that, you don't need to use `--project` constantly.

2. Both `--new` and `--update` options are optional. They're there to prevent the user from doing something they don't want to do. I will explain in an example:

```
$   neptune notebook sync --project piotr/sandbox my-notebook.ipynb
$   mv my-notebook.ipynb my-notebook-with-a-different-name.ipynb 
$   neptune notebook sync --project piotr/sandbox my-notebook-with-a-different-name.ipynb 
ERROR: Notebook path changed since last checkpoint from

    /home/piotr/neptune-experiments/my-notebook.ipynb

to

    /home/piotr/neptune-experiments/my-notebook-with-a-different-name.ipynb

Use following command if you want to create new notebook

    neptune notebook sync --new --project piotr/sandbox my-notebook-with-a-different-name.ipynb

or following command if you want to update existing notebook

    neptune notebook sync --update --project piotr/sandbox my-notebook-with-a-different-name.ipynb

```

Reply:
I think a similar output appears when you're trying to sync a new checkpoint to a notebook that's not yours.

Reply:
Oh, I didn't know.

In that case we can add a note that those are optional and can be skipped.

--new -> if notebook isn't known to Neptune it will be created anyways

--update -> if notebook is know to Neptune it will be updated anyways

Reply:
We'll fix it with @moshekruger. Thanks @pitercl and @jakubczakon for the example and additional elucidations.